dashboard/app/controllers/peer_reviews_controller.rb

```diff
@@ -21,7 +21,7 @@ def dashboard
 
     @course_unit_map = {}.tap do |course_unit_map|
       plc_courses.each do |course|
-        course_unit_map[course.id] = course.plc_course_units.map {|course_unit| [course_unit.name, course_unit.id]}
+        course_unit_map[course.id] = course.plc_course_units.select {|course_unit| !course_unit.deprecated?}.map {|course_unit| [course_unit.name, course_unit.id]}
```

Comment:
Kind of a long line, might be nice to break up

Reply:
tbh, I tried to do this and our style rules prevented me from committing until I put it all back on one line! Is there a way to sidestep those?

Reply:
Huh, interesting... 🤷 I got no offenses detected with:
```
      plc_courses.each do |course|
        course_unit_map[course.id] = course.plc_course_units.
          select {|course_unit| !course_unit.deprecated?}.
          map {|course_unit| [course_unit.name, course_unit.id]}
      end
```
Maybe you need the periods on the end of each line? If that doesn't work, forget it :).